mitodl/ocw-to-hugo
# Convert videos to markdown, instead of relying on the hugo videogallery layout

Blocked by https://github.com/mitodl/hugo-course-publisher/pull/274 (should be merged first so that the markdown generated in this PR will render properly)

#### Pre-Flight checklist


#### What are the relevant tickets?
Related to https://github.com/mitodl/hugo-course-publisher/issues/182

#### What's this PR do?
Generates markdown for videos, and no longer sets the `layout` to `videogallery`.  
This will be dependent on changes in hugo-course-publisher as well.

Also removes unused sections of the test master json files to try to prevent intermittent timeout errors during the `scanCourses` tests.

#### How should this be manually tested?
- Create a `course.json` file containing this:
  ```
  {
    "courses": [
        "ec-711-d-lab-energy-spring-2011",
        "18-06-linear-algebra-spring-2010",
        "8-04-quantum-physics-i-spring-2013"
    ]
  }
  ```

- Run `ocw-to-hugo -c course.json -i input - o output` (Add the `--download` option if necessary)
- The content of `output/courses/ec-711-d-lab-energy-spring-2011/sections/intro-energy-basics-human-power/_index.md` should include this:
```
---
{{< video-gallery-item href="/courses/ec-711-d-lab-energy-spring-2011/sections/intro-energy-basics-human-power/lecture-1-introduction-to-energy" section="Week 1: Introduction, Energy Basics & Human Power" title="Lecture 1: Introduction to Energy" description="Description: This lecture introduces fundamental energy concepts: energy around in the world, energy units, a quick electricity review, and some estimation practice activities. The session ends with a syllabus overview. Speaker: Amy Banzaert" thumbnail="https://img.youtube.com/vi/SbpeBF8D_m4/default.jpg" >}}
{{< video-gallery-item href="/courses/ec-711-d-lab-energy-spring-2011/sections/intro-energy-basics-human-power/lab-1-human-power" section="Week 1: Introduction, Energy Basics & Human Power" title="Lab 1: Human Power" description="Description: This lab consists of three parts: Water pumping, with a PVC hand pump and a cement rocker pump Water carrying, with a tump line, head carry, hand carry, and Q-drum Woodshop training Speaker: Amy Banzaert, Amit Gandhi" thumbnail="https://img.youtube.com/vi/7MzwxhtVfFc/default.jpg" >}}
```

If you copy the output over to `hugo-course-publisher/site/content/courses`, [on this branch](https://github.com/mitodl/hugo-course-publisher/pull/274), and start it up, there should be no errors and the relevant video gallery pages should display correctly.
